DXRando/DeusEx/Classes/DXRandoCrowdControlLink.uc

```diff
@@ -680,47 +696,54 @@ function bool isCrowdControl( string msg) {
         return False;
     }
     
-    if (InStr(msg,"}")!=Len(msg)-1){
-        //PlayerMessage("Message doesn't end with curly");
+    //Explicitly check last character of string to see if it's a closing curly
+    tmp = Mid(msg,Len(msg)-1,1);
+    //if (InStr(msg,"}")!=Len(msg)-1){
+    if (tmp != "}"){
+        //PlayerMessage("Message doesn't end with curly.  Ends with '"$tmp$"'.");
         return False;    
     }
     
     //Check to see if it looks like it has the right fields in it
     
     //id field
     if (InStr(msg,"id")==-1){
+        //PlayerMessage("Doesn't have id");
         return False;
     }
     
     //code field
     if (InStr(msg,"code")==-1){
+        //PlayerMessage("Doesn't have code");
         return False;
     }    
     //viewer field
     if (InStr(msg,"viewer")==-1){
+        //PlayerMessage("Doesn't have viewer");
         return False;
     }   
     //type field
     if (InStr(msg,"type")==-1){
+        //PlayerMessage("Doesn't have type");
         return False;
     }
     
     //Check to see if there are multiple messages stuck together
```

Comment:
Have we verified if this happens or not? It's LAN traffic so I don't think this should happen normally, and it'll auto retry anyways